packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1429,6 +1428,26 @@ export class ContainerRuntime
 
 		ReportOpPerfTelemetry(this.context.clientId, this.deltaManager, this.logger);
 		BindBatchTracker(this, this.logger);
+
+		this.mc.logger.sendTelemetryEvent({
+			eventName: "Config",
+			options: JSON.stringify(runtimeOptions),
+			compressionDisabled: this.mc.config.getBoolean(
```

Comment:
If possible, we should read various feature gate flags only once in code (and store it in variables) - this reduceds substantially bundle size and reduces risks of mis-synchronization of code.

Reply:
Agree on the bundle size but the feature flags are immutable (after the first read they're going to be cached) which was an intentional design decision to rule out any issues related to synchronization.